PowerPointLabs/Test/UnitTest/HighlightLab/BaseHighlightLabTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using Test.Util;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace Test.UnitTest.HighlightLab
+{
+    [TestClass]
+    public class BaseHighlightLabTest : BaseUnitTest
```

Comment:
Ah I thought it was convention since the PositionsLab and HighlightLab unit tests all have a base file which all the other test files extend. Should I remove it? It might be helpful if in future we write more highlight lab tests.